src/Compilers/CSharp/Test/Syntax/Parsing/SeparatedSyntaxListParsingTests.cs

```diff
@@ -205,7 +206,7 @@ class C
 }
 
 class M<,> { }
-");
+", options: TestOptions.Regular.WithTuplesFeature());
```

Comment:
It feels like it would make sense to have similar test with tuples disabled. That should be the behavior with language version 6. The same comment applies to other parsing tests.